src/coordinator/users.spec.js

```diff
@@ -850,4 +850,135 @@ describe('Coordinator/Users', function() {
       });
     });
   });
+
+  describe('subscribeEvent', function() {
+    context('when all the required parameters are provided', function() {
+      let expectedUserEvent;
+      let promise;
+      let request;
+      let event;
+      let user;
+      let userEventFromServer;
+      let toCamelCase;
+
+      beforeEach(function() {
+        event = fixture.build('event');
+        user = fixture.build('contxtUser');
+
+        expectedUserEvent = fixture.build('contxtUserEvent');
+        userEventFromServer = fixture.build(
+          'contxtUserEvent',
+          expectedUserEvent,
+          {
+            fromServer: true
+          }
+        );
+
+        request = {
+          ...baseRequest,
+          post: sinon.stub().resolves(userEventFromServer)
+        };
+        toCamelCase = sinon
+          .stub(objectUtils, 'toCamelCase')
+          .callsFake((app) => expectedUserEvent);
+
+        const users = new Users(baseSdk, request, expectedHost);
+        promise = users.subscribeEvent(user.id, event.id);
+      });
+
+      it('posts the user event to the server', function() {
+        expect(request.post).to.be.calledWith(
+          `${expectedHost}/users/${user.id}/events/${event.id}`
+        );
+      });
+
+      it('formats the returned user event', function() {
+        return promise.then(() => {
+          expect(toCamelCase).to.be.calledWith(userEventFromServer);
+        });
+      });
+
+      it('returns a fulfilled promise', function() {
+        return expect(promise).to.be.fulfilled;
```

Comment:
Should include that it returns the subscription in the description and change this to `return expect(promise).to.be.fulfilled.and.to.eventually.equal(expectedSubscription)`